frontend/src/pages/Interventions/FirstActivity.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { ReactElement }  from "react";
+import styled from "styled-components";
+import Layout from "../../components/Layout";
+import PromptSpeech from "../../components/PromptSpeech";
+import ReplayButton from '../../components/ReplayButton';
+import PurpleButton from "../../components/PurpleButton";
+import { SKY } from "../../constants/colors";
+
+interface FirstActivityProps {
+  title: string,
+  imageUrl: string,
+}
+
+const Container = styled.div`
+  display: flex;
+  justify-content: center;
+`;
+
+const MainContent = styled.div`
+  @media (max-width: 900px) {
+    width: 100%;
+  }
+`;
+
+const DescriptionText = styled.p`
+  color: #666;
+  font-family: "Rubik";
+  font-weight: 700;
+  font-size: 18px;
+  margin-bottom: 10px;
+`;
+
+const WordTitle = styled.p`
+  font-family: "Rubik";
+  font-size: 35px;
+  font-weight: 700;
```

Comment:
Should we make components that already have the fonts correct, so that we all use the same components, and then changing the font will change it for everyone?

Reply:
As in like for the font size?

Reply:
I think it's okay to change the size instead of making the component. If it's the font-family, I think im just putting it to enforce it cuz i think in the index.js file rubik should be the one that's first found

Reply:
ahhh makes sense nvm then